tools/accuracy_checker/accuracy_checker/preprocessor/audio_preprocessing.py

```diff
@@ -65,19 +65,40 @@ def parameters(cls):
         parameters = super().parameters()
         parameters.update({
             'duration': NumberField(
-                value_type=int, min_value=0,
+                value_type=float, min_value=0, optional=True, default=None,
                 description="Length of audio clip in seconds."
             ),
+            'duration_in_samples': NumberField(
+                value_type=int, min_value=0, optional=True, default=None,
+                description="Length of audio clip in samples."
+            ),
             'max_clips': NumberField(
-                value_type=int, min_value=1, default=1,
+                value_type=int, min_value=1, default=999, optional=True,
```

Comment:
it is hard to understand, 999 was set by user or auto value
what if somebody have too long input?
I suggest to use -1 as default or not set at all for understanding that user set nothing

Reply:
done